android/app/src/main/java/io/parity/signer/MainActivity.java

```diff
@@ -19,13 +19,6 @@ protected String getMainComponentName() {
         return "NativeSigner";
     }
 
-    @Override
-    protected void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        getWindow().setFlags(WindowManager.LayoutParams.FLAG_SECURE,
-                           WindowManager.LayoutParams.FLAG_SECURE);
-    }
-
```

Comment:
Why we re-enable screenshot functionality of the app? Is it an expected feature?

Reply:
Ooops, forgot about that, I'm removing this part every time I want to share a screenshot with you guys because I'm on a real phone. Super annoying, looking forward for https://github.com/paritytech/parity-signer/issues/230